e2e/app/component/card/data-resource-card.ts

```diff
@@ -84,8 +84,13 @@ export default class DataResourceCard extends BaseCard {
     }
 
     const dataTableRows = await datatable.getRowCount();
+
     if (dataTableRows >= 1) {
-      return datatable;
+      const firstRow = await datatable.getRow(1);
```

Comment:
Our datatables used to be blank when no filters matched:
![image](https://user-images.githubusercontent.com/24514630/212165943-91e34053-221b-44e8-8f9f-833cf0d7c3de.png)
But now they show an empty message which is a row:
![image](https://user-images.githubusercontent.com/24514630/212166134-5a99c7fd-f00c-42eb-b36e-e359e3e8fbdc.png)

We can customize the text of this message, if we would like.